docs/source/initialsetup/compilingmame.rst

```diff
@@ -275,9 +280,18 @@ Mojave or later for Intel Macs or macOS 11.0 Big Sur for Apple Silicon. You will
 
 Next you’ll need to get SDL2 installed.
 
-* Go to `this site <http://libsdl.org/download-2.0.php>`_ and download the *Mac OS X* .dmg file
-* If the .dmg doesn’t auto-open, open it
-* Click “Macintosh HD” (your Mac’s hard disk) in the Locations sidebar of a **Finder** window, then open the **Library** folder and drag the **SDL2.framework** folder from the SDL disk image into the **Frameworks** folder. You will have to authenticate with your user password.
+* Go to `this site <http://libsdl.org/download-2.0.php>`_ and download the *macOS* .dmg file
+* If the .dmg doesn't auto-open, open it
+* Click 'Macintosh HD' (or whatever your Mac's hard disk is named) in the left pane of a **Finder** window, then open the **Library** folder and drag the **SDL2.framework** folder from the SDL disk image into the **Frameworks** folder. You will have to authenticate with your user password.
```

Comment:
This line didn’t get hard-wrapped with the other lines.

Reply:
Interesting. I'd tried splitting lists before and found Sphinx wasn't taking that. Now it's working fine. Well, I'll have a fix for this in today. I just want to be absolutely sure it compiles correctly first.

Reply:
Unrelated:
When looking at the self-build packages list: 
```
    pacman -Syu
    pacman -S curl git make
    pacman -S mingw-w64-x86_64-gcc mingw-w64-x86_64-libc++ mingw-w64-x86_64-lld mingw-w64-x86_64-python
    pacman -S mingw-w64-x86_64-SDL2 mingw-w64-x86_64-SDL2_ttf
    pacman -S mingw-w64-x86_64-qt5
    pacman -S mingw-w64-i686-gcc mingw-w64-i686-libc++ mingw-w64-i686-lld mingw-w64-i686-python
    pacman -S mingw-w64-i686-SDL2 mingw-w64-i686-SDL2_ttf
    pacman -S mingw-w64-i686-qt5
```

w64 is obviously the toolchain being 64-bit, and I'm assuming that the i686 set are for building 32-bit binaries from the 64-bit tools. Should those be removed from the documentation, or should we still mention it as potentially possible but not officially supported build?

Reply:
You can hard-wrap a line in a list and have multiple paragraphs in a list item – I do it all the time.  There are examples in https://raw.githubusercontent.com/mamedev/mame/mame0231/docs/source/techspecs/layout_files.rst

The key thing to remember is that you need to indent the lines following the initial line of the item, and avoid blank lines between items.  Here’s a sample copied straight out of that file:
```
* Components may be conditionally drawn depending on the element’s state by
  supplying ``state`` and/or ``statemask`` attributes.  If present, these
  attributes must be non-negative integers.  If only the ``state`` attribute is
  present, the component will only be drawn when the element’s state matches its
  value.  If only the ``statemask`` attribute is present, the component will
  only be drawn when all the bits that are set in its value are set in the
  element’s state.

  If both the ``state`` and ``statemask`` attributes are present, the component
  will only be drawn when the bits in the element’s state corresponding to the
  bits that are set in the ``statemask`` attribute’s value match the value of the
  corresponding bits in the ``state`` attribute’s value.

  (The component will always be drawn if neither ``state`` nor ``statemask``
  attributes are present, or if the ``statemask`` attribute’s value is zero.)
* Each component may have a ``bounds`` child element specifying its position and
  size (see :ref:`layfile-concepts-coordinates`).  If no such element is
  present, the bounds default to a unit square (width and height of 1.0) with
  the top left corner at (0,0).

  A component’s position and/or size may be animated according to the element’s
  state by supplying multiple ``bounds`` child elements with ``state``
  attributes.  The ``state`` attribute of each ``bounds`` child element must be
  a non-negative integer.  The ``state`` attributes must not be equal for any
  two ``bounds`` elements within a component.

  If the element’s state is lower than the ``state`` value of any ``bounds``
  child element, the position/size specified by the ``bounds`` child element
  with the lowest ``state`` value will be used.  If the element’s state is
  higher than the ``state`` value of any ``bounds`` child element, the
  position/size specified by the ``bounds`` child element with the highest
  ``state`` value will be used.  If the element’s state is between the ``state``
  values of two ``bounds`` child elements, the position/size will be
  interpolated linearly.
* Each component may have a ``color`` child element specifying an RGBA colour
  (see :ref:`layfile-concepts-colours` for details).  This can be used to
  control the colour of geometric, algorithmically drawn, or textual components.
  For ``image`` components, the colour of the image pixels is multiplied by the
  specified colour.  If no such element is present, the colour defaults to
  opaque white.

  A component’s color may be animated according to the element’s state by
  supplying multiple ``color`` child elements with ``state`` attributes.  The
  ``state`` attributes must not be equal for any two ``color`` elements within a
  component.

  If the element’s state is lower than the ``state`` value of any ``color``
  child element, the colour specified by the ``color`` child element with the
  lowest ``state`` value will be used.  If the element’s state is higher than
  the ``state`` value of any ``color`` child element, the colour specified by
  the ``color`` child element with the highest ``state`` value will be used.  If
  the element’s state is between the ``state`` values of two ``color`` child
  elements, the RGBA colour components will be interpolated linearly.
```

Reply:
As for i686, I think I already mentioned some of the pitfalls.  How actively do we want to recommend against it?